--- automake-1.10.2/m4/depout.m4.orig	2008-10-26 16:02:41.000000000 +0100
+++ automake-1.10.2/m4/depout.m4	2008-12-27 11:47:51.502510478 +0100
@@ -12,7 +12,8 @@
 # _AM_OUTPUT_DEPENDENCY_COMMANDS
 # ------------------------------
 AC_DEFUN([_AM_OUTPUT_DEPENDENCY_COMMANDS],
-[# Autoconf 2.62 quotes --file arguments for eval, but not when files
+[{
+# Autoconf 2.62 quotes --file arguments for eval, but not when files
 # are listed without --file.  Let's play safe and only enable the eval
 # if we detect the quoting.
 case $CONFIG_FILES in
@@ -60,6 +61,7 @@
     echo '# dummy' > "$dirpart/$file"
   done
 done
+}
 ])# _AM_OUTPUT_DEPENDENCY_COMMANDS
 
 
