--- automake-1.7/automake.in.wiget	Wed Sep 25 19:54:22 2002
+++ automake-1.7/automake.in	Thu Sep 26 09:42:12 2002
@@ -31,7 +31,7 @@
 
 BEGIN
 {
-  my $perllibdir = $ENV{'perllibdir'} || '@datadir@/@PACKAGE@-@APIVERSION@';
+  my $perllibdir = $ENV{'perllibdir'} || '@datadir@/@PACKAGE@';
   unshift @INC, $perllibdir;
 }
 
@@ -126,7 +126,7 @@
 # VERSION as string so that eg version 0.30 will print correctly.
 my $VERSION = '@VERSION@';
 my $PACKAGE = '@PACKAGE@';
-my $libdir = '@datadir@/@PACKAGE@-@APIVERSION@';
+my $libdir = '@datadir@/@PACKAGE@';
 
 # Some regular expressions.  One reason to put them here is that it
 # makes indentation work better in Emacs.
--- automake-1.7/aclocal.in.wiget	Thu Sep 19 22:34:47 2002
+++ automake-1.7/aclocal.in	Thu Sep 26 09:43:59 2002
@@ -29,7 +29,7 @@
 
 BEGIN
 {
-  my $perllibdir = $ENV{'perllibdir'} || '@datadir@/@PACKAGE@-@APIVERSION@';
+  my $perllibdir = $ENV{'perllibdir'} || '@datadir@/@PACKAGE@';
   unshift @INC, $perllibdir;
 }
 
@@ -185,7 +185,7 @@
     # Search the versioned directory near the end, and then the
     # unversioned directory last.  Only do this if the user didn't
     # override acdir.
-    push (@dirlist, "$acdir-$APIVERSION")
+    push (@dirlist, "$acdir")
 	if $acdir eq $default_acdir;
 
     # By default $(datadir)/aclocal doesn't exist.  We don't want to
