diff -durN -x '*~' -x '*.orig' automake-1.8.orig/lib/config.sub automake-1.8/lib/config.sub
--- automake-1.8.orig/lib/config.sub	2003-11-25 23:07:32.000000000 +0100
+++ automake-1.8/lib/config.sub	2004-01-02 21:29:53.839623027 +0100
@@ -380,6 +380,9 @@
 	amd64)
 		basic_machine=x86_64-pc
 		;;
+	amd64-*)
+		basic_machine=x86_64-`echo $basic_machine | sed 's/^[^-]*-//'`
+		;;
 	amdahl)
 		basic_machine=580-amdahl
 		os=-sysv
