--- ./m4/depout.m4.org	Mon May 14 02:01:09 2001
+++ ./m4/depout.m4	Mon Apr 22 18:37:18 2002
@@ -12,8 +12,8 @@
 test x"$AMDEP_TRUE" != x"" ||
 for mf in $CONFIG_FILES; do
   case "$mf" in
-  Makefile) dirpart=.;;
-  */Makefile) dirpart=`echo "$mf" | sed -e 's|/[^/]*$||'`;;
+  Makefile|GNUmakefile) dirpart=.;;
+  */Makefile|*/GNUmakefile) dirpart=`echo "$mf" | sed -e 's|/[^/]*$||'`;;
   *) continue;;
   esac
   grep '^DEP_FILES *= *[^ #]' < "$mf" > /dev/null || continue
--- ./automake.in.org	Thu Aug 23 07:26:53 2001
+++ ./automake.in	Mon Apr 22 18:36:10 2002
@@ -1940,7 +1940,9 @@
         }
 
 	# Let the language do some special magic if required.
-	$lang->target_hook ($aggregate, $object, $full);
+	if (defined $lang) {
+		$lang->target_hook ($aggregate, $object, $full);
+	}
 
 	if ($derived_source)
 	{
