--- automake-1.9.6/lib/config.sub~	2005-07-10 17:16:04.000000000 +0000
+++ automake-1.9.6/lib/config.sub	2006-02-12 14:02:11.000000000 +0000
@@ -836,6 +836,12 @@
 		;;
 	ppc-*)	basic_machine=powerpc-`echo $basic_machine | sed 's/^[^-]*-//'`
 		;;
+	ppc74[05]0)
+		basic_machine=powerpc-unknown
+		;;
+	ppc74[05]0-*)
+		basic_machine=powerpc-`echo $basic_machine | sed 's/^[^-]*-//'`
+		;;
 	ppcle | powerpclittle | ppc-le | powerpc-little)
 		basic_machine=powerpcle-unknown
 		;;
