--- automake-1.15.1/lib/config.sub.orig	2017-06-20 16:43:35.596510193 +0200
+++ automake-1.15.1/lib/config.sub	2017-06-20 17:09:44.756492278 +0200
@@ -1037,6 +1037,12 @@
 	ppc-* | ppcbe-*)
 		basic_machine=powerpc-`echo $basic_machine | sed 's/^[^-]*-//'`
 		;;
+	ppc74[05]0)
+		basic_machine=powerpc-unknown
+		;;
+	ppc74[05]0-*)
+		basic_machine=powerpc-`echo $basic_machine | sed 's/^[^-]*-//'`
+		;;
 	ppcle | powerpclittle)
 		basic_machine=powerpcle-unknown
 		;;
