diff -uNr automake-1.6.3/automake.in automake-1.6.3.new/automake.in
--- automake-1.6.3/automake.in	Wed Aug 28 19:41:13 2002
+++ automake-1.6.3.new/automake.in	Wed Aug 28 19:41:47 2002
@@ -3017,10 +3017,7 @@
 	}
 	if (basename ($onelib) !~ /$libname_rx$/)
 	{
-	    # FIXME this should only be a warning for foreign packages
-	    # FIXME should put line number here.  That means mapping
-	    # from library name back to variable name.
-	    &am_error ("`$onelib' is not a standard libtool library name");
+	    warn ("`$onelib' is not a standard libtool library name");
 	}
 
 	if (variable_defined ($xlib . '_LIBADD'))
